tools/FakeItEasy.Build/Program.cs

```diff
@@ -81,22 +75,20 @@ public static void Main(string[] args)
                     action: testDirectory => RunTests(testDirectory));
             }
 
-            Target("get-version", () => version = Read(ToolPaths.GitVersion, "/showvariable NuGetVersionV2"));
-
             Target(
-                "pack-projects",
-                DependsOn("build", "outputDirectory", "pdbgit", "get-version"),
+                "pack",
+                DependsOn("build", "outputDirectory", "pdbgit"),
                 forEach: ProjectsToPack,
-                action: project => Run("dotnet", $"pack {project} --configuration Release --no-build --output {OutputDirectory} /p:Version={version}"));
+                action: project => Run("dotnet", $"pack {project} --configuration Release --no-build --output {OutputDirectory}"));
 
-            Target(
-                "pack-nuspecs",
-                DependsOn("outputDirectory", "get-version"),
-                () => Run(ToolPaths.NuGet, $"pack {AnalyzerMetaPackageNuspecPath} -Version {version} -OutputDirectory {OutputDirectory} -NoPackageAnalysis"));
+            ////Target(
+            ////    "pack-nuspecs",
+            ////    DependsOn("outputDirectory", "get-version"),
+            ////    () => Run(ToolPaths.NuGet, $"pack {AnalyzerMetaPackageNuspecPath} -Version {version} -OutputDirectory {OutputDirectory} -NoPackageAnalysis"));

```

Comment:
There are ways of making this work without using nuget.exe and passing the version explicitly, but for the purpose of this test, I just commented this out. Also, IIRC you were considering getting rid of this meta-package.